image/sigmoid/test.py

```diff
@@ -0,0 +1,12 @@
+import cv2
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver does.